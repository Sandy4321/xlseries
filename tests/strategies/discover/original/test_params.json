{"alignment": "vertical",
 "blank_rows": false,
 "composed_headers": false,
 "continuity": true,
 "data_ends": 256,
 "data_starts": 2,
 "frequency": "M",
 "headers_coord": ["B1", "C1-C2"],
 "missings": true,
 "missing_value": null,
 "progressive_aggregation": false,
 "series_names": null,
 "time_composed": false,
 "time_multicolumn": false,
 "time_alignment": 0,
 "time_format": "datetime.datetime",
 "time_header": false,
 "time_header_coord": "A1",
 "time_header_next_to_data": null}
