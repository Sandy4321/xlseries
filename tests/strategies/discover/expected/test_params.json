{"alignment": ["vertical", "vertical", "vertical"],
 "blank_rows": ["False", "False", "False"],
 "composed_headers": ["False", "False", "False"],
 "continuity": ["True", "True", "True"],
 "data_ends": [256, 256, 256],
 "data_starts": [2, 2, 2],
 "frequency": ["M", "M", "M"],
 "headers_coord": ["B1", "C1", "C2"],
 "missings": ["True", "True", "True"],
 "missing_value": ["None", "None", "None"],
 "multifrequency": ["False", "False", "False"],
 "progressive_aggregation": ["False", "False", "False"],
 "series_names": ["None", "None", "None"],
 "time_composed": ["False", "False", "False"],
 "time_multicolumn": ["False", "False", "False"],
 "time_alignment": [0, 0, 0],
 "time_format": ["datetime.datetime", "datetime.datetime", "datetime.datetime"],
 "time_header": ["False", "False", "False"],
 "time_header_coord": ["A1", "A1", "A1"],
 "time_header_next_to_data": ["None", "None", "None"]}
